### PR TITLE
Transform to date from input datetime-local value

### PR DESCRIPTION
**Pull request by VictorADS**
This change is to support ZonedDateTime for our entities

### CODE REVIEW

File: src/service/date-util.service.ts [hunk 1]
```diff
@@ -65,9 +65,9 @@ export class DateUtils {
     dateformat () {
         return this.pattern;
     }
-    //TODO Change this method when moving from datetime-local input to NgbDatePicker
+    // TODO Change this method when moving from datetime-local input to NgbDatePicker
     toDate(date: any): Date {
```
Comment (gmarziou) [inline src/service/date-util.service.ts]: Should `date` be of type `string` rather than `any`?

Comment (VictorADS) [inline src/service/date-util.service.ts]: Changing `date` to `string` implies that `dateParts` is of type `string[]`
Thus we have to use `parseInt(string, radix)` everytime we use `dateParts[i]`.

Should I go back to `any` or do I make the change ?

Comment (deepu105) [inline src/service/date-util.service.ts]: its fine lets not forget the benefits of JS :smiling_imp:

**Comment by deepu105** [comment]
im changing it to any

**Comment by deepu105** [comment]
new version released

**Pull request merged**